docs/jupiterone.md

```diff
@@ -1,13 +1,15 @@
 # Integration with JupiterOne
 
-## {{provider}} + JupiterOne Integration Benefits
+## Mimecast + JupiterOne Integration Benefits
 
 TODO: Iterate the benefits of ingesting data from the provider into JupiterOne.
 Consider the following examples:
```

Comment:
I realize you are probably going to come back to the docs, but think you can go ahead and remove this